src/views/Editor.vue

```diff
@@ -159,7 +160,22 @@ watch(proposalData, () => {
           title: 'Title'
         }"
       />
-      <div class="s-base mb-5">
+      <div class="flex">
+        <Link
+          :is-active="!previewEnabled"
+          text="Editor"
```

Comment:
How about using same naming than GitHub? Editor is the whole page not just the body
```suggestion
          text="Write"
```